src/java.base/share/classes/java/lang/invoke/MethodHandleNatives.java

```diff
@@ -697,6 +697,22 @@ static boolean canBeCalledVirtual(MemberName symbolicRef, Class<?> definingClass
                 symbolicRefClass.isInterface());                     // Mdef implements Msym
     }
 
+    //--- AOTCache support
```

Comment:
Please add the imports (even though they are unused) from upstream that were added in be095e78c3f87a3a6ef0746d0ec55bf14ff55e5c.

Reply:
Restored the imports.